.github/workflows/weekly.yml

```diff
@@ -45,11 +41,14 @@ jobs:
           jdk: '8, 11, 17, 21',
           branch: main
         }, {
-          flink: 1.17.2,
-          branch: v3.1
+          flink: 1.18.1,
+          branch: v3.2
+        }, {
+          flink: 1.19.0,
+          branch: v3.2,
+          jdk: '8, 11, 17, 21',
```

Comment:
nit: it would be nice to order the `flink` and `branch` in the list the same way. (both increasing down / both decreasing down)